js/three-orbit-controls.min.js

```diff
@@ -0,0 +1 @@
multiplyScalar(s.panSpeed),k(y.x,y.y),H.copy(v),s.update()}(e)}}function G(e){!1!==s.enabled&&(document.removeEventListener("mousemove",B,!1),document.removeEventListener("mouseup",G,!1),s.dispatchEvent(l),E=m.NONE)}function K(e){!1===s.enabled||!1===s.enableZoom||E!==m.NONE&&E!==m.ROTATE||(e.preventDefault(),e.stopPropagation(),s.dispatchEvent(u),function(e){e.deltaY<0?D(N()):e.deltaY>0&&Y(N()),s.update()}(e),s.dispatchEvent(l))}function W(e){!1!==s.enabled&&!1!==s.enableKeys&&!1!==s.enablePan&&function(e){var t=!1;switch(e.keyCode){case s.keys.UP:k(0,s.keyPanSpeed),t=!0;break;case s.keys.BOTTOM:k(0,-s.keyPanSpeed),t=!0;break;case s.keys.LEFT:k(s.keyPanSpeed,0),t=!0;break;case s.keys.RIGHT:k(-s.keyPanSpeed,0),t=!0}t&&(e.preventDefault(),s.update())}(e)}function q(e){if(!1!==s.enabled){switch(e.preventDefault(),e.touches.length){case 1:switch(s.touches.ONE){case THREE.TOUCH.ROTATE:if(!1===s.enableRotate)return;V(e),E=m.TOUCH_ROTATE;break;case THREE.TOUCH.PAN:if(!1===s.enablePan)return;z(e),E=m.TOUCH_PAN;break;default:E=m.NONE}break;case 2:switch(s.touches.TWO){case THREE.TOUCH.DOLLY_PAN:if(!1===s.enableZoom&&!1===s.enablePan)return;!function(e){s.enableZoom&&X(e),s.enablePan&&z(e)}(e),E=m.TOUCH_DOLLY_PAN;break;case THREE.TOUCH.DOLLY_ROTATE:if(!1===s.enableZoom&&!1===s.enableRotate)return;!function(e){s.enableZoom&&X(e),s.enableRotate&&V(e)}(e),E=m.TOUCH_DOLLY_ROTATE;break;default:E=m.NONE}break;default:E=m.NONE}E!==m.NONE&&s.dispatchEvent(u)}}function Q(e){if(!1!==s.enabled)switch(e.preventDefault(),e.stopPropagation(),E){case m.TOUCH_ROTATE:if(!1===s.enableRotate)return;_(e),s.update();break;case m.TOUCH_PAN:if(!1===s.enablePan)return;Z(e),s.update();break;case m.TOUCH_DOLLY_PAN:if(!1===s.enableZoom&&!1===s.enablePan)return;!function(e){s.enableZoom&&I(e),s.enablePan&&Z(e)}(e),s.update();break;case m.TOUCH_DOLLY_ROTATE:if(!1===s.enableZoom&&!1===s.enableRotate)return;!function(e){s.enableZoom&&I(e),s.enableRotate&&_(e)}(e),s.update();break;default:E=m.NONE}}function J(e){!1!==s.enabled&&(s.dispatchEvent(l),E=m.NONE)}function $(e){!1!==s.enabled&&e.preventDefault()}s.domElement.addEventListener("contextmenu",$,!1),s.domElement.addEventListener("mousedown",F,!1),s.domElement.addEventListener("wheel",K,!1),s.domElement.addEventListener("touchstart",q,!1),s.domElement.addEventListener("touchend",J,!1),s.domElement.addEventListener("touchmove",Q,!1),s.domElement.addEventListener("keydown",W,!1),-1===s.domElement.tabIndex&&(s.domElement.tabIndex=0),this.update()},THREE.OrbitControls.prototype=Object.create(THREE.EventDispatcher.prototype),THREE.OrbitControls.prototype.constructor=THREE.OrbitControls,THREE.MapControls=function(e,t){THREE.OrbitControls.call(this,e,t),this.mouseButtons.LEFT=THREE.MOUSE.PAN,this.mouseButtons.RIGHT=THREE.MOUSE.ROTATE,this.touches.ONE=THREE.TOUCH.PAN,this.touches.TWO=THREE.TOUCH.DOLLY_ROTATE},THREE.MapControls.prototype=Object.create(THREE.EventDispatcher.prototype),THREE.MapControls.prototype.constructor=THREE.MapControls;
```

Comment:
Are you 100% sure this is the 0.112.1 version as suggested here? https://github.com/arrowtype/recursive-minisite/pull/54

Reply:
Not sure. Will check with that PR!